wrappers/python/nntile/__init__.py

```diff
@@ -11,5 +11,8 @@
 # @author Aleksandr Mikhalev
 # @date 2023-02-22
```

Comment:
Add yourself to the list of authors of the file and update data

Reply:
I think that `git` does it better than human.